### PR TITLE
Feature/add netstat by instance

### PR DESCRIPTION
**Pull request by dothebart**
### Scope & Purpose

measure the ports the SUT allocates.

- [x] :hammer: test infrastructure

### CODE REVIEW

File: js/client/modules/@arangodb/testutils/result-processing.js [hunk 1]
```diff
@@ -664,6 +669,34 @@ function unitTestTabularPrintResults (options, results, otherResults) {
...
 // / @brief creates a chartlist of the longest running tests
 // //////////////////////////////////////////////////////////////////////////////
+function summarizeStats(deltaStats) {
+  let sumStats = {};
+  for (let instance in deltaStats) {
+    sumStats['netstat'] = {};
+    if (instance === 'netstat') {
+      for (let nsInstance in deltaStats[instance]) {
+        for (let j = 0; j < 2; j ++) {
+          let dir = ['in', 'out'][j];
```
Comment (adityamukho) [inline js/client/modules/@arangodb/testutils/result-processing.js:679]:
```suggestion
        for (const dir of ['in', 'out']) {
```

**Review by adityamukho — COMMENTED** [review]
Just one small syntactic suggestion. Otherwise LGTM.

**Review by adityamukho — APPROVED** [review]
LGTM

File: LICENSES-OTHER-COMPONENTS.md [hunk 2]
```diff
@@ -1918,6 +1928,16 @@ License: https://raw.githubusercontent.com/arangodb/arangodb/devel/js/node/node_
 License Name: MIT License
 License Id: MIT
 
+
+#### node-netstat
+
+Name: node-netstat
+Version: 1.8.0 + adjustments for ArangoDB
```
[the remark this answers is not in the document]
Comment (dothebart) [reply LICENSES-OTHER-COMPONENTS.md:1935]: thanks.

**Pull request merged**